docs/docs.trychroma.com/markdoc/content/sidebar-config.ts

```diff
@@ -158,9 +158,23 @@ const sidebarConfig: AppSection[] = [
     icon: SquareTerminalIcon,
     pages: [
       { id: "install", name: "Installing the CLI" },
-      { id: "run", name: "Run a Chroma Server" },
       { id: "sample-apps", name: "Sample Apps" },
```

Comment:
should sample apps be in the command section? maybe under "install"? and then we will have a separate sample apps section altogether